zerver/lib/bugdown/__init__.py

```diff
@@ -754,11 +754,14 @@ def handleMatch(self, match):
 def make_emoji(codepoint, display_string):
     # type: (Text, Text) -> Element
     src = '/static/generated/emoji/images/emoji/unicode/%s.png' % (codepoint,)
+    # Replace underscore in emoji's title with space
+    title = display_string[1:-1].replace("_", " ")
+
     elt = markdown.util.etree.Element('img')
     elt.set('src', src)
     elt.set('class', 'emoji')
     elt.set("alt", display_string)
```

Comment:
from [this](https://silktide.com/i-thought-title-text-improved-accessibility-i-was-wrong/) post and a few others, `alt` should be the one to be set, as it's the one used by screen readers.